cmd/livepeer/livepeer.go

```diff
@@ -513,11 +513,14 @@ func main() {
 			// Set price per pixel base info
 			if *pixelsPerUnit <= 0 {
 				// Can't divide by 0
-				panic(fmt.Errorf("The amount of pixels per unit must be greater than 0, provided %d instead\n", *pixelsPerUnit))
+				panic(fmt.Errorf("-pixelsPerUnit must be > 0, provided %d", *pixelsPerUnit))
+			}
+			if !isFlagSet["pricePerUnit"] && *pricePerUnit == 0 {
+				// Prevent orchestrators from unknowingly providing free transcoding
+				panic(fmt.Errorf("-pricePerUnit must be set"))
 			}
-			if *pricePerUnit <= 0 {
-				// Prevent orchestrator from unknowingly provide free transcoding
-				panic(fmt.Errorf("Price per unit of pixels must be greater than 0, provided %d instead\n", *pricePerUnit))
+			if *pricePerUnit < 0 {
+				panic(fmt.Errorf("-pricePerUnit must be > 0, provided %d", *pricePerUnit))
```

Comment:
```suggestion
				panic(fmt.Errorf("-pricePerUnit must be >= 0, provided %d", *pricePerUnit))
```
Even though most public Os won't ever set it = 0, I still think we should keep the error msg close to the truth for some future dev reading the error msg or reading the codebase and make it clear that it can be 0?

Reply:
Fixed in a81e82b